great_expectations/rule_based_profiler/config/base.py

```diff
@@ -521,32 +521,34 @@ def __init__(
         super().__init__(commented_map=commented_map)
 
     @classmethod
-    def get_config_class(cls) -> Type["RuleBasedProfilerConfig"]:  # noqa: F821
-        return cls
-
-    @classmethod
-    def get_schema_class(cls) -> Type["RuleBasedProfilerConfigSchema"]:  # noqa: F821
-        return RuleBasedProfilerConfigSchema
-
-    @classmethod
-    def from_commented_map(
-        cls, commented_map: CommentedMap
-    ) -> "RuleBasedProfilerConfig":
+    def from_commented_map(cls, commented_map: CommentedMap):  # type: ignore[no-untyped-def]
```

Comment:
Same note throughout - I think `# type: ignore` should be used very sparingly.

Reply:
I'd prefer `Any` over this tbh.

Reply:
@cdkini The `Any` here is not a great fit either.  Let us discuss what we can do.